### PR TITLE
Blog post for MMTF format support

### PR DESCRIPTION
**Pull request by richardjgowers**

### CODE REVIEW

File: _posts/2016-11-23-Support-for-MMTF.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+title: Support for MMTF has arrived!
+---
+
+The upcoming 0.16.0 release of MDAnalysis will have
+[support for MMTF](https://twitter.com/mmtf_spec/status/799704395046760448)!
+[MMTF](http://mmtf.rcsb.org/) is a new format designed to provide compact, efficient and fast browsing of the [Protein
+Data Bank](www.rcsb.org/).
+Support for MMTF within MDAnalysis is offered through reading locally stored MMTF files,
+or through fetching the file directly from the PDB archive through the new
+`MDAnalysis.fetch_mmtf` function.
```
Comment (kain88-de) [inline _posts/2016-11-23-Support-for-MMTF.md]: Maybe you can add that the mmtf files are very small and will be quick to download also on slow connections.

Comment (richardjgowers) [inline _posts/2016-11-23-Support-for-MMTF.md]: ok done

**Comment by kain88-de** [comment]
You can also write how to preview the current release. I could update my nightly build after all mmtf stuff is merged.

**Comment by richardjgowers** [comment]
Yeah if we could do a conda build that would be good.  Installing from source is a little tedious to recommend.

**Comment by kain88-de** [comment]
I can update my limited nightly build again. It has some limitations but better then nothing.

File: _posts/2016-11-23-Support-for-MMTF.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+```
+
+The performance of loading MMTF files is a large improvement over traditional ascii PDB files,
+with the above system of approximately 2.4M atoms taking under 10 seconds to load.
+The compressed format and [efficient algorithms](https://github.com/rcsb/mmtf/blob/v1.0/spec.md)
+for storing the data mean that downloading structures will also require much less bandwidth,
+making this possible even on slow connections.
+
+MMTF files can support many different models for a given structure and this is made available
+through the `.models` attribute of a MDAnalysis Universe.  This provides a list of AtomGroup
```
Comment (richardjgowers) [inline _posts/2016-11-23-Support-for-MMTF.md]: I doesn't work currently.  You could maybe add a kwarg to mmtf to define a single model to load, so you can put a trajectory on it, so `mda.Universe('this.mmtf', 'traj.trr', model=4)`

**Comment by orbeckst** [comment]
Is this good to go? 

From the discussion above it seems that you already agreed not to put anything in there along the lines of "try the development branch". That's totally fine with me, the blog post is a good teaser for 0.16.0. I would make it go live now.

File: _posts/2016-11-23-Support-for-MMTF.md (hunk 2 above)
Comment (orbeckst) [inline _posts/2016-11-23-Support-for-MMTF.md]: Can you load the MMTF and then use `Universe.load_new()`?

Comment (richardjgowers) [inline _posts/2016-11-23-Support-for-MMTF.md]: You could only do this if the number of atoms was the same in each.  So If the mmtf has multiple models, probably not.

**Comment by richardjgowers** [comment]
I added a link to the dev install instructions,  @kain88-de merge if you're happy

**Pull request merged**